src/spiffs_check.c

```diff
@@ -161,7 +161,13 @@ static s32_t spiffs_delete_obj_lazy(spiffs *fs, spiffs_obj_id obj_id) {
     return SPIFFS_OK;
   }
   SPIFFS_CHECK_RES(res);
-  u8_t flags = 0xff & ~SPIFFS_PH_FLAG_IXDELE;
+  u8_t flags = 0xff;
```

Comment:
Why do the flags need to be initialized at all? Doesn't the _spiffs_rd always fill the value in (or return an error)?

Reply:
you are right, this is not really necessary. amended.